packages/social-metadata-previews/tests/twitter/TwitterImageTest.js

```diff
@@ -0,0 +1,180 @@
+/* External dependencies */
+import React from "react";
+import renderer from "react-test-renderer";
+
+/* Internal dependencies */
+import TwitterImage from "../../src/twitter/TwitterImage";
+import delayComponentSnapshot from "./testHelpers/delayComponentSnapshot";
+import { LANDSCAPE_HEIGHT,
```

Comment:
When splitting over multiple lines, all imports should go on a newline.

```suggestion
import {
	LANDSCAPE_HEIGHT,
```

Reply:
Corrected.